scripts.js

```diff
@@ -5,6 +5,15 @@ var newGameBtn = document.getElementById('js-newGameButton'),
     pickScissors = document.getElementById('js-playerPick_scissors'),
     name = document.getElementById('js-playerName');
 
+//function who don't allow more than 15 letters in name  
+function lenghtName () {
+   var playerName = player.name.length;
+    if (playerName >= 15){
+        alert('Your name is to long');
+        newGame();
```

Comment:
you start again newGame but you need to remember about line 66 - there still you're checking that player.name exist and run a game, no matter of result in this function